evergreen/config.yml

```diff
@@ -899,6 +910,7 @@ buildvariants:
   - name: benchmarks
   - name: long-running-tests
   - name: swift-build-and-test
+  - name: test-on-exfat
```

Comment:
do you want to test this on arm64 too?

Reply:
I am fine with it only running on one MacOS builder, the goal is just to test this filesystem type and I just picked this Mac builder randomly. I think if there are future problems on exFAT, then just this one should be enough to catch them.